### PR TITLE
Postmortem for today's boskos outage

### PR DESCRIPTION
**Pull request by Katharine**
Excitement!

### CODE REVIEW

File: docs/post-mortems/2019-03-26.md [hunk 1]
```diff
@@ -0,0 +1,312 @@
...
+At around 17:20, several hours before the outage, we started upgrading the Prow clusters from Kubernetes
+1.11 to 1.13. The build cluster has 150 nodes, and each update takes five minutes, so for this entire
+process to complete would take about ten hours. Several hours later, nothing looked amiss, and we
+went to bed.
+
+At 23:04 the node running the only Boskos pod was drained to be replaced with a 1.13 node, and boskos
+was moved to a new node. At this point it used the StatefulSet's spec, which specified a newer image
+version with the bug. As a result, boskos was never able to come up on the new node.
+
+## Detection
```
Comment (fejta) [inline docs/post-mortems/2019-03-26.md]: https://github.com/kubernetes/test-infra/issues/11953?

Comment (Katharine) [reply docs/post-mortems/2019-03-26.md]: Added a link to the AIs.

**Pull request merged**